testing/test_config.py

```diff
@@ -352,6 +352,25 @@ def test_inifilename(self, tmpdir):
         assert config.inicfg.get('name') == 'value'
         assert config.inicfg.get('should_not_be_set') is None
 
+    @pytest.mark.issue1618
+    def test_consider_plugin(self, testdir):
+        pytest.importorskip('xdist')
```

Comment:
This test failed in CI, but only on `xdist` tox environments: I think we should run this in all environments because while I understand this was initially reported and diagnosed with xdist, the problem is general to pytest.